client/src/App.jsx

```diff
@@ -27,7 +28,9 @@ function App() {
   return (
     //Use BrowserRouter to use Link from react-router-dom
     <BrowserRouter>
+      <BackgroundImage bgImage={bgImage} />
```

Comment:
Why didn't you name your component BgImage? :) 
I mean why not using the whole name for you property?